homeworks/max.snischuk_snischuk/PokemonProject/calculator.html

```diff
@@ -47,6 +47,51 @@ <h1 class="calculator__title">Calculate pokemons</h1>
                 <p class="calculator__time" data-math-time></p>
             </div>
         </main>
+        <footer class="footer">
```

Comment:
Same content jumps I can see on this page, when both inputs are filled and I hit `=` button